lua/weapons/arccw_base/cl_hud.lua

```diff
@@ -211,6 +216,8 @@ function SWEP:DrawHUD()
 
             if alpha > 0 then
 
+				angpos.Pos = angpos.Pos - Vector(0,0,5) - EyeAngles():Right() * 4
```

Comment:
sometimes larger weapons will obscure it
and it can get in the way of what the player's tryna look at